activesupport/lib/active_support/core_ext/yaml.rb

```diff
@@ -0,0 +1,17 @@
+require "yaml"
+
+module YAML
```

Comment:
If that is the case we can do it, but right now I prefer duplication. This is not something that I personally want to spread to more yaml files. I also don't want to give users access to this feature so as harder it is to do, better for us.